src/components/Thread.vue

```diff
@@ -15,12 +15,14 @@
 					</div>
 				</div>
 			</div>
-			<ThreadEnvelope v-for="env in thread"
-				:key="env.databaseId"
-				:envelope="env"
-				:mailbox-id="$route.params.mailboxId"
-				:expanded="expandedThreads.includes(env.databaseId)"
-				@toggleExpand="toggleExpand(env.databaseId)" />
+			<div class="mail-thread-envelopes">
```

Comment:
could you elaborate what this wrapping div does?

Reply:
I guess just for the sake of styling, right?

Reply:
Yes, it is responsible for the scrollbar.

Reply:
If we just scroll the envelopes we can omit the present `display: sticky` hack on the thread header.

Reply:
This div is gone now (due to reverting the design changes).